build.rs

```diff
@@ -0,0 +1,116 @@
+#[macro_use]
+extern crate lazy_static;
+
+use std::fmt;
+use std::path::Path;
+use std::fs::{File, read_dir, metadata};
+use std::io::{Write, BufWriter};
+
+// Location of the dist folder
+#[cfg(not(target_os = "windows"))]
+const DIST: &str = "frontend/dist/";
+#[cfg(target_os = "windows")]
+const DIST: &str = "frontend\\dist\\";
+
+// Where to write the Assets out to
+#[cfg(not(target_os = "windows"))]
+const ASSETOUT: &str = "src/asset.in";
+#[cfg(target_os = "windows")]
+const ASSETOUT: &str = "src\\asset.in";
+
+// Ignore directories specified here
+#[cfg(target_os = "windows")]
+lazy_static! {
+    static ref IGNORE: Vec<&'static str> = {
+        vec![
+            "frontend\\dist\\ember-fetch"
+        ]
+    };
+}
+#[cfg(not(target_os = "windows"))]
+lazy_static! {
+    static ref IGNORE: Vec<&'static str> = {
+        vec![
+            "frontend/dist/ember-fetch"
```

Comment:
@steveklabnik I've confirmed that the ember dir causes the whole thing to fail documenting code in the example crate though I'm unsure why. We might want to keep this available for now (until that issue is fixed somehow) or just have it check against that path.